src/Eccube/EventListener/TwigInitializeListener.php

```diff
@@ -59,20 +59,25 @@ public function __construct(
      */
     public function onKernelRequest(GetResponseEvent $event)
     {
-        $globals = $this->twig->getGlobals();
-        if (array_key_exists('BaseInfo', $globals) && $globals['BaseInfo'] === null) {
-            $this->twig->addGlobal('BaseInfo', $this->baseInfoRepository->get());
-        }
-
-        if (!$event->isMasterRequest()) {
-            return;
+        try {
```

Comment:
`$client->disableReboot()` is no longer required by #2775. So this try-catch is not required too.

Reply:
Test same case on web was successfully, but failed on phpunit.maybe 
Kernel was re-booted on each request, then service container was re-initialize . so keep $client->disableReboot() in testcase

Reply:
Which test?